### ISSUE TITLE
System.IO.IOException: Error loading native library "C:\Users\Administrator\Desktop\servers\server1\grpc_csharp_ext.x64.dll".  when runs at x64

### ISSUE THREAD

**Issue by luchunminglu** [grpc/grpc#21550]
### What version of gRPC and what language are you using?
2.26.0

### What operating system (Linux, Windows,...) and version?
windows server 2016 datacenter

### What runtime / compiler are you using (e.g. python version or version of gcc)
.net4.5.2

### What did you do?
when runs at x86 32bit, it's fine.
but at x64 64bit, server.start() throws System.IO.IOException: Error loading native library "C:\Users\Administrator\Desktop\servers\server1\grpc_csharp_ext.x64.dll".  
```

var server = new Server(rpcOptions)
                {
                    Services = { HealthCheckService.BindService(new HealthCheckServiceImpl()), MonitorService.BindService(new MonitorServiceImpl()), HpcsService.BindService(new HpcsServiceImpl())},
                    Ports = { new ServerPort(Constants.BindIp, Constants.ServerPort, ServerCredentials.Insecure) }
                };

server.Start();
```

System.IO.IOException: Error loading native library "C:\Users\Administrator\Desktop\servers\server1\grpc_csharp_ext.x64.dll". 
   在 Grpc.Core.Internal.UnmanagedLibrary..ctor(String[] libraryPathAlternatives)
   在 Grpc.Core.Internal.NativeExtension.LoadUnmanagedLibrary()
   在 Grpc.Core.Internal.NativeExtension.LoadNativeMethods()
   在 Grpc.Core.Internal.NativeExtension..ctor()
   在 Grpc.Core.Internal.NativeExtension.Get()
   在 Grpc.Core.GrpcEnvironment.GrpcNativeInit()
   在 Grpc.Core.GrpcEnvironment..ctor()
   在 Grpc.Core.GrpcEnvironment.AddRef()
   在 Grpc.Core.Server..ctor(IEnumerable`1 options)

**Comment by chandmk** [comment]
In my case it is caused by required MSVCP140.DLL.  Make sure that VisualC++ Redistributables are available on servers (>= 2017)

**Comment by JimHume** [comment]
I have tried installing the VC++ redistributables but haven't had success in getting around this issue.
I've tried versions 10.0.30319.01, and 10.0.40219, and I cannot get 2.26 to load properly on a 32-bit machine. Is there a specific version of the 2010 redist's that should be used?

This effectively breaks the usage of grpc outright for us unfortunately.

**Comment by chandmk** [comment]
This worked
https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads

**Comment by JimHume** [comment]
@chandmk Thank you for the quick turn-around. 

I don't know why I defaulted to thinking it was a 2010 redist. The all-encompassing VC++ redist did indeed work as described for 32-bit--thanks for that.

**Comment by jdennison-iforium** [comment]
This was also an issue for me after upgrading to 2.26.0 on .Net 4.6.1. Is it documented somewhere that the VisualC++ Redistributables are required (and at what version)?

**Comment by JimHume** [comment]
This was just encountered from a different angle: "The program can't start because api-ms-win-crt-runtime-| 1-1-0.dll is missing from your computer. Try reinstalling the program to fix this problem."

Is this fix included in 1.27 via this PR: https://github.com/grpc/grpc/pull/21810 ?

This was quite brutal to see in production--it's especially an issue with air-gapped PC's that don't perform windows update.

**Comment by steveoh** [comment]
This is an issue for me on windows server 2016 with a dot net core 3.1 application. I have a lot of the c++ redistributables...

![image](https://user-images.githubusercontent.com/325813/75399438-79919700-58b9-11ea-8831-e3ac0bb63304.png)

**Comment by jtattermusch** [comment]
This should be fixed in 2.27+ release thanks to https://github.com/grpc/grpc/pull/21792 and #21810
